.github/workflows/sast.yml

```diff
@@ -1,5 +1,10 @@
 name: SAST
 
+concurrency:
+  # for PR's cancel the running task, if another commit is pushed
```

Comment:
This is great! 🙌

Reply:
Thanks 😊 ! But I need to give credit where credit is due: @Tieske 👏 
https://github.com/Kong/lua-resty-healthcheck/pull/142#discussion_r1358181243